types/ckeditor__ckeditor5-list/src/checktodolistcommand.d.ts

```diff
@@ -1,9 +1,27 @@
 import { Command } from '@ckeditor/ckeditor5-core';
 import { Element } from '@ckeditor/ckeditor5-engine';
 
+/**
+ * The check to-do command.
+ *
+ * The command is registered by the {@link module:list/todolistediting~TodoListEditing} as
+ * the `checkTodoList` editor command and it is also available via aliased `todoListCheck` name.
+ */
 export default class CheckTodoListCommand extends Command {
-    value: Element[];
+    /**
+     * A list of to-do list items selected by the {@link module:engine/model/selection~Selection}.
+     */
+    get value(): Element[];
+    protected set value(value: Element[]);
+
+	/**
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/ckeditor__ckeditor5-list/src/checktodolistcommand.d.ts;
  Line: 17;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.